packages/forms/src/widgets/DatalistWidget/DatalistWidget.scss

```diff
@@ -3,6 +3,8 @@ $tf-typeahead-items-background-color: $white !default;
 $tf-typeahead-items-focus-background-color: shade($tf-typeahead-items-background-color, 5) !default;
 $tf-typeahead-items-shadow-color: $shadow !default;
 $tf-typeahead-no-result-color: $silver !default;
+$tf-typeahead-section-header-color: #63AEBD !default;
+$tf-typeahead-section-border-color: #E0E0E0 !default;
```

Comment:
Could you pull those colors out of the theme?

Reply:
sorry, I don't get this...do you mean move these to talend-bootstrap-theme?